helm/prometheus-rules/templates/alerting-rules/elasticsearch.rules.yml

```diff
@@ -1,3 +1,4 @@
+{{- if eq .Values.managementCluster.provider.flavor "vintage" }}
```

Comment:
Are we officially not supporting ES anymore on newer clusters?

Reply:
We deprecated it 1 year ago, so yes we will not alert it on capi. It;s only used on kvm anyway